### DISCUSSION TITLE
Bewegungsmelder einbinden

### DISCUSSION THREAD

**Discussion by edgarsperling** [LeonardSchwenk/public-gruppenadressen-generator#41] [Q&A] [answered]
Hallo,
erstmal vielen Dank für die viele Arbeit die Du schon in das Projekt gesteckt hat.
Es hilft mir grade sehr bei meinem Projekt.

Was ich nicht als Objekt gefunden habe ist ein Bewegungsmelder.
Wie binde ich den am besten ein? Kann ich dafür auch einen Präsenzmelder nehmen?

Gruß Edgar

**Comment by sebastiantheobald** [comment, marked as answer]
Hi @edgarsperling - das für die netten Worte :)! 
Stand jetzt würde ich die Präsenzmelder empfehlen ja. Alternativ kannst du auch einfach das Objekt individuell benennen: 
<img width="651" alt="image" src="https://github.com/LeonardSchwenk/public-gruppenadressen-generator/assets/23461821/b58ec04e-4fff-414b-a137-657f11ce1a26">

Künftig werden wir hier noch mehr Vorschläge einbauen - vorerst solltest du damit aber weiterkommen :)! Ich hoffe ich kann dir damit helfen!
